src/components/CallView/CallView.vue

```diff
@@ -66,6 +66,34 @@
 									:call-participant-model="callParticipantModel"
 									:shared-data="sharedDatas[shownRemoteScreenPeerId]"
 									:is-big="true" />
+								<!-- presenter overlay -->
+								<TransitionWrapper v-if="callParticipantModel.attributes.peerId === shownRemoteScreenPeerId && shouldShowPresenterOverlay
+										&& callParticipantModel.attributes.videoAvailable "
+									:key="'video-overlay-wrapper' + callParticipantModel.attributes.peerId"
+									name="slide-down">
+									<VideoVue :key="'video-overlay-' + callParticipantModel.attributes.peerId"
+										class="viewer-overlay__video"
+										:token="token"
+										:model="callParticipantModel"
+										:shared-data="sharedDatas[shownRemoteScreenPeerId]"
+										:show-talking-highlight="false"
+										is-presenter-overlay
+										un-selectable
+										hide-bottom-bar
+										@click-video="toggleShowPresenterOverlay" />
+								</TransitionWrapper>
+								<!-- presenter button when it is collapsed -->
+								<NcButton v-else-if="!showPresenterOverlay && callParticipantModel.attributes.videoAvailable"
+									:key="'video-overlay-collapsed' + callParticipantModel.attributes.peerId"
+									:aria-label="t('spreed', 'Show presenter')"
+									class="presentar-overlay--collapse"
```

Comment:
```suggestion
									class="presenter-overlay--collapse"
```

Reply:
I'd also add a `title` here, as it's not clear, what button does
![image](https://github.com/nextcloud/spreed/assets/93392545/3841b206-ea7c-418a-88dc-99ba1ff305e9)